apps/els_lsp/test/els_hover_SUITE.erl

```diff
@@ -524,10 +549,32 @@ nonexisting_module(Config) ->
     ?assertEqual(Expected, Result),
     ok.
 
+%%==============================================================================
+%% Helpers
+%%==============================================================================
+
+mock_eep48_render_fail() ->
+    %% els_docs calls the render functions twice
+    Fail4 = fun(_, _, _, _) -> {error, {mocked, ?MODULE, ?LINE}} end,
+    Fail3 = fun(_, _, _) -> {error, {mocked, ?MODULE, ?LINE}} end,
+    meck:expect(els_eep48_docs, render, Fail4),
+    meck:expect(els_eep48_docs, render, Fail3),
+    meck:expect(els_eep48_docs, render_type, Fail4),
+    meck:expect(els_eep48_docs, render_type, Fail3).
+
 has_eep48_edoc() ->
     list_to_integer(erlang:system_info(otp_release)) >= 24.
+
 has_eep48(Module) ->
     case catch code:get_doc(Module) of
-        {ok, _} -> true;
-        _ -> false
+        {ok, {docs_v1, _, erlang, _, _, _, Docs}} ->
+            lists:any(
+                fun
+                    ({_, _, _, #{}, _}) -> true;
+                    ({_, _, _, _, _}) -> false
+                end,
+                Docs
+            );
```

Comment:
This and `has_eep48/1` in the other suite were only updated to know when the cases can be skipped. If skipping is not wanted this can go away (or be used to improve test debugging).